HPXMLtoOpenStudio/resources/hpxml_schematron/EPvalidator.xml

```diff
@@ -18,7 +18,7 @@
   <sch:pattern>
     <sch:title>[Building]</sch:title>
     <sch:rule context='/h:HPXML/h:Building'>
-      <sch:assert role='ERROR' test='count(h:Site/h:Address) &lt;= 1'>Expected 0 or 1 element(s) for xpath: Building/Site/Address</sch:assert> <!-- See [Address] -->
+      <sch:assert role='ERROR' test='count(h:Site) &lt;= 1'>Expected 0 or 1 element(s) for xpath: Building/Site</sch:assert> <!-- See [BuildingSite] -->
       <sch:assert role='ERROR' test='count(h:ProjectStatus/h:EventType) = 1'>Expected 1 element(s) for xpath: Building/ProjectStatus/EventType</sch:assert>
       <sch:assert role='ERROR' test='count(h:BuildingDetails) = 1'>Expected 1 element(s) for xpath: Building/BuildingDetails</sch:assert> <!-- See [BuildingDetails] -->
```

Comment:
You didn't introduce this, but I just realized that all of the error messages in this section should be changed from:
`Expected ... for xpath: Building/Foo/...`
to:
`Expected ... for xpath: Foo/...`

That is, the error messages should be relative to the context, which already includes the Building element (`<sch:rule context='/h:HPXML/h:Building'>`).